tests/framework/clients/rancher/config.go

```diff
@@ -6,6 +6,7 @@ type Config struct {
 	Host       string `yaml:"host"`
 	AdminToken string `yaml:"adminToken"`
 	Insecure   *bool  `yaml:"insecure" default:"true"`
+	Cleanup    *bool  `yaml:"cleanup" default:"true"`
```

Comment:
If I set the cleanup option to `false` would I end up with all the provisioned clusters in Rancher?
What else are we cleaning up here, users?

I just wanted to understand this part, this PR looks good for the scope defined.

Reply:
If you set the cleanup option to `false` it would not clean anything up. So the clusters would be there, as well as the users.